metricbeat/docs/modules/http.asciidoc

```diff
@@ -42,5 +42,9 @@ The following metricsets are available:
 
 * <<metricbeat-metricset-http-json,json>>
 
+* <<metricbeat-metricset-http-server,server>>
```

Comment:
I think http server metricset is spurious previous to a rename to graphite? also present here: https://github.com/elastic/beats/pull/4734/files#diff-8303434bf263f758175d97b66f919690R48